src/styles.scss

```diff
@@ -1,18 +1,20 @@
 @use "@material/theme" with (
-    $on-surface: #000,
-    $primary: #3F51B5, // indigo 500
-    $on-primary: #fff,
-    $secondary: #2196F3, // blue 500
-    $on-secondary: #fff,
-    $background: #fff,
+  $on-surface: #000,
```

Comment:
you introduce formatting changes in the same commit you add new features. Please split it (so the formatting is in a separate commit)